webview/src/plots/components/ribbon/Ribbon.tsx

```diff
@@ -23,12 +22,6 @@ export const Ribbon: React.FC = () => {
   const revisions = useSelector(
     (state: PlotsState) => state.webview.selectedRevisions
   )
-  const [order, setOrder] = useState<string[]>([])
-  const reorderId = 'id'
-
-  useEffect(() => {
-    setOrder(pastOrder => performOrderedUpdate(pastOrder, revisions, reorderId))
-  }, [revisions])
```

Comment:
[F] There was a race condition between the data update and `useEffect` that would cause `reorderObjectList(order, revisions, reorderId)` to drop selected checkpoint experiments from the ribbon. Deleting this means that we need to update the message passed for comparison plots and split the order of the comparison table columns from `selectedRevisions` (to reinstate => `'should not reorder the ribbon when comparison plots are reordered'`).

Reply:
I will do this as an immediate follow up